components/http_foundation.rst

```diff
@@ -517,13 +517,18 @@ call::
         'immutable'        => true,
         'last_modified'    => new \DateTime(),
         'etag'             => 'abcdef',
+        'terminate_on_cache_hit' => false,
```

Comment:
We need to revert the changes in this document. The option is not part of the HTTP cache headers but only supported by our `HttpCache` class coming from the HttpKernel component.